src/test/e2e/utils/helpers.ts

```diff
@@ -260,9 +260,7 @@ export const e2e = test
 						"--skip-welcome",
 						"--skip-release-notes",
 						`--user-data-dir=${userDataDir}`,
-						`--extensions-dir=${extensionsDir}`,
 						`--install-extension=${path.join(E2ETestHelper.CODEBASE_ROOT_DIR, "dist", "e2e.vsix")}`,
-						`--extensionDevelopmentPath=${E2ETestHelper.CODEBASE_ROOT_DIR}`,
```

Comment:
remove the development paths for e2e test to ensure all e2e tests are run with vsix packaged for production